tendrl/node_agent/manager/manager.py

```diff
@@ -99,12 +99,14 @@ def __init__(self, machine_id):
                     is None:
                 utils.delete_local_node_context()
 
+        node_id = utils.get_local_node_context()
         super(
             NodeAgentManager,
             self
         ).__init__(
             "node",
-            utils.get_local_node_context(),
+            node_id,
+            node_id,
```

Comment:
I understand the logic for integartion_id but in case of node_agent for the time being we are using node_id as integration_id and in case of gluster/ceph inetgartion cluster_id is used as integration_id. Also as the base Manager class needs both `integration_id` and `node_id` as constructor argument, so at the moment passing node_id for both.